docs/content/configuration/coordinator.md

```diff
@@ -22,6 +22,7 @@ The coordinator node uses several of the global configs in [Configuration](../co
 
 |Property|Description|Default|
 |--------|-----------|-------|
+|`druid.coordinator.beOverlord`|Boolean value for whether this coordinator node should act like an overlord as well. This configuration allows users to simplify a druid cluster by not having to deploy any standalone overlord nodes. If set to true, then make sure `druid.selectors.indexing.serviceName` is correctly set to coordinator service name everywhere and all the overlord configuration must be added to the coordinator node configuration.|true|
```

Comment:
considering, user needs to take explicit step to make things work, I am thinking of actually keeping `beOverlord` false by default and rather test it on our clusters first. @gianm @fjy what do you think?